trivy-image-scan/action.yml

```diff
@@ -92,4 +92,11 @@ runs:
       uses: github/codeql-action/upload-sarif@v2
       if: ${{ (success() || failure()) && inputs.output-mode == 'github' }}
       with:
-        sarif_file: ${{ env.OUTPUT }}
\ No newline at end of file
+        sarif_file: ${{ env.OUTPUT }}
+    
+    - name: unset env variables
+      shell: bash
```

Comment:
This would need to run `if: ${{ always() }}`

Another alternative you could consider is not using env variables at all but instead using step ouputs.That would avoid the need to unset at the end.

Reply:
i have changed the env variables with prefix TRIVY_